lib/source-check.js

```diff
@@ -0,0 +1,29 @@
+'use strict'
+
+const fs = require('fs')
+
+function checkForTranspiledCode (filename) {
+  const readFile = fs.readFileSync(filename, 'utf8')
+  const regex = /function\s+(\w+)/g
+  let matchedObj
+  let isTranspiled = false
+
+  // Check for a source map
+  isTranspiled = readFile.includes('//# sourceMappingURL=')
+
+  while ((matchedObj = regex.exec(readFile)) !== null) {
+    // Avoid infinite loops with zero-width matches
+    if (matchedObj.index === regex.lastIndex) {
+      regex.lastIndex++
+    }
+    // Loop through results and check length of fn name
+    matchedObj.forEach((match, groupIndex) => {
+      if (groupIndex !== 0 && match.length < 3) {
+        isTranspiled = true
```

Comment:
Considering it transpiled if just one function name is short is maybe a bit too strict. For example, the `assert` module contains an `ok` function, but it's not transpiled.

Reply:
I think this is kinda the same as the other comment. You want me to set isTranspiled to false if detected a longer function during the while loop?

Reply:
It wasn't intended to be the same as the other comment, but that's not a bad idea :smile: